spec/browsers/air_traffic_framework_user_agent_spec.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+
+require 'user_agent'
+
+describe UserAgent::Browsers::AirTrafficFramework do
+  let(:user_agent) { UserAgent.parse(ua_string) }
+
+  shared_examples 'an ATC browser' do |version = nil|
+    it { expect(user_agent.browser).to eql 'atc' }
+    it { expect(user_agent).not_to be_bot }
+
+    if version
+      it { expect(user_agent.version.to_s).to eql version }
+    else
+      it { expect(user_agent.version).to be_nil }
+    end
+  end
+
+  shared_examples 'a browser on unknown OS' do
+    it { expect(user_agent.os).to be_nil}
+  end
+
+  shared_examples 'a browser on Apple Watch / watchOS' do |os_version|
```

Comment:
Minor: maybe to make `os_version` default to `nil` and remove all the `nil` arguments below. Same for the below shared_examples.

Reply:
Done! Good callout.